assets/src/data/eventespresso/core/resolvers/relations.js

```diff
@@ -67,8 +67,9 @@ export function* getRelatedEntities(
 	if ( entity.isNew ) {
 		return DEFAULT_EMPTY_ARRAY;
 	}
-	const modelName = entity.modelName.toLowerCase();
+	relationModelName = singularModelName( relationModelName );
 	const pluralRelationName = pluralModelName( relationModelName );
+	const modelName = entity.modelName.toLowerCase();
 	const relationResourceProperty = pluralRelationName + 'Resource';
```

Comment:
here's another plural that escaped the purge, although this one looks more legit

Reply:
see two lines up.